contracts/test/integration/IntegrationTestPriceBoundPSM.t.sol

```diff
@@ -114,15 +114,17 @@ contract IntegrationTestPriceBoundPSMTest is DSTest {
     }
 
     /// @notice PSM is set up correctly and view functions are working
-    function testGetRedeemAmountOut() public {
+    function testGetRedeemAmountOut(uint256 amountVoltIn) public {
+        if (amountVoltIn > mintAmount) return;
```

Comment:
instead of doing this, either do `vm.assume(mintAmount >= amountVoltIn);`, or preferably choose a uint size that is under mintAmount so that amountVoltIn never violates that assumption. It's better to move the uint size down because then forge won't have a bunch of discarded runs.